hydra_agent/hydra_graph.py

```diff
@@ -55,7 +55,7 @@ def get_endpoints(self,api_doc, redis_connection):
             alias="Entrypoint",
             properties=entrypoint_properties)
         self.redis_graph.add_node(entrypoint_node)
-        redis_connection.set("EntryPoint", entrypoint_properties)
+        redis_connection.set("EntryPoint", json.dumps(entrypoint_properties))
```

Comment:
@PrajwalM2212  I think, we don't need to have that line of code because we already have a node for the same. So, Can you try to run without this line? And don't forget to send the screenshot.